src/api/sso/Dockerfile

```diff
@@ -1,47 +1,64 @@
-## Base ########################################################################
-# Use a larger node image to do the build for native deps (e.g., gcc, pytyhon)
-FROM node:lts as base
+## Base ###########################################################################
+# Set up the base layer
+FROM node:16-alpine3.15 as base
 
 # Reduce npm log spam and colour during install within Docker
 ENV NPM_CONFIG_LOGLEVEL=warn
 ENV NPM_CONFIG_COLOR=false
 
+RUN npm i -g pnpm
+
 # We'll run the app as the `node` user, so put it in their home directory
 WORKDIR /home/node/app
-# Copy the package.json and lock file over
-COPY package*.json /home/node/app/
+
+## Dependencies-dev ###############################################################
```

Comment:
I don't think we're actually using this pattern anywhere else, so maybe we should remove this.